### DISCUSSION TITLE
Slack link not working

### DISCUSSION THREAD

**Discussion by MBRasmussen** [dataplat/dbatools#7263] [General]
Hey
Was trying to join on Slack but the link to get an invite listed on https://dbatools.io/contact/ does not seem to be active any more.
Don't know if anyone relevant will see this, but couldn't find any other places to report this
![image](https://user-images.githubusercontent.com/73524273/116045312-37591680-a672-11eb-9286-e062aecde24f.png)

**Comment by D0rKn1Gh7** [comment]
I'm waiting on this to be resolved as well

**Reply by Ant-Green** [reply]
Believe this is one for @potatoqualitee to fix.

Chrissy this is happening in the dbatools.io/slack link and the GitHub chat links when posting bugs etc.

**Reply by Ant-Green** [reply]
In the mean time you can use my invite link for SQLSlack

https://join.slack.com/t/sqlcommunity/shared_invite/zt-poklchwh-qOND8KOs2jdHu04yzADn0w

**Reply by potatoqualitee** [reply]
Thank you both so much! I have updated it with a permanent (I hope) link. Previously, it was limited to 30 days but this time it said never expires 🤞 LPlease let me know if there are any issues.

**Reply by dbdave** [reply]
Morning! any chance anyone has an invite link I can use?  can't find any that work

thank you!

**Reply by Ant-Green** [reply]
Temporary link

https://join.slack.com/t/sqlcommunity/shared_invite/zt-1wyl7m437-IbynnRS~t9gHEIPnWzEsAA

**Reply by dbdave** [reply]
in, thank you!

**Comment by Ant-Green** [comment]
Can use my personal link for the time being

Join me on Slack – it’s a faster, simpler way to work. Sign up here, from any device: https://join.slack.com/t/sqlcommunity/shared_invite/zt-1wyl7m437-IbynnRS~t9gHEIPnWzEsAA


________________________________
